ocis-pkg/config/helpers.go

```diff
@@ -31,14 +31,23 @@ var (
 func DefaultConfigSources(filename string, drivers []string) []string {
 	var sources []string
 
-	for i := range defaultLocations {
-		dirFS := os.DirFS(defaultLocations[i])
+	locations := []string{}
+	if os.Getenv("OCIS_CONFIG_DIR") != "" {
```

Comment:
Would be nicer to store `os.Getenv("OCIS_CONFIG_DIR")` in a var instead of calling it twice, but not critical
```suggestion
	if v := os.Getenv("OCIS_CONFIG_DIR"); v != "" {
	    locations = append(locations, v)
```